### ISSUE TITLE
What are all the reserved keywords for Column names in SequelizeJS

### ISSUE THREAD

**Issue by rhughes89** [sequelize/sequelize#8150]
I just ran into an issue where I couldn't make define a column called "Model" as reported here:

 https://github.com/sequelize/sequelize/issues/1948

I was wondering if anyone knows any other restrictions in terms of model names that people would need to be aware of

**Comment by dhessler** [comment]
"attributes" also seems to cause problems, would be great to have a list of the reserved column names.

**Comment by jpreese** [comment]
I'm pretty sure I'm running into this problem with "Enabled". Sequelize tells me it cannot find a column:

> SQLITE_ERROR: table My_Table has no column named Enabled

But if I change the name, to say Enabled_Work, it's fine.

**Comment by tbopp** [comment]
I have also ran into the exact same problem using: '_reducedName_' as a column name. It works, however, when I changed the column name to: '_Reduced_Name_'. 

I've reviewed the list of reserved words for both SQL and MySQL (I'm using workbench) and I've been unable to locate a word that even closely resembles my original column name. This leads me to believe it's an issue with sequelize. 

Documentation on reserved words for column names in sequelize would be helpful. - Thanks
